### PR TITLE
fc::time_point_sec::to_iso_string is broken #597

### PR DESCRIPTION
**Pull request by cogutvalera**
PR for https://github.com/bitshares/bitshares-core/issues/597

### CODE REVIEW

File: tests/time_test.cpp [hunk 1]
```diff
@@ -38,8 +38,11 @@ BOOST_AUTO_TEST_CASE(time_point_sec_test)
 
     time_point_sec tp3g(0xc0000000U);
     // commented next 2 tests as they will only work with boost >= 1.64
-    //BOOST_CHECK_EQUAL( "2072-01-28T16:51:12", tp3g.to_iso_string() );
```
Comment (jmjatlanta) [inline tests/time_test.cpp]: Please remove (now inaccurate) comment above.

Successfully tested with Boost 1.65.

**Review by jmjatlanta — CHANGES_REQUESTED** [review]
Minor change requested. See below.

File: tests/time_test.cpp (hunk 1 above)
Comment (cogutvalera) [reply tests/time_test.cpp]: Thanks !

File: tests/time_test.cpp [hunk 2]
```diff
@@ -37,9 +37,10 @@ BOOST_AUTO_TEST_CASE(time_point_sec_test)
     BOOST_CHECK_EQUAL( "20380119T031408", tp2g.to_non_delimited_iso_string() );
 
     time_point_sec tp3g(0xc0000000U);
-    // commented next 2 tests as they will only work with boost >= 1.64
-    //BOOST_CHECK_EQUAL( "2072-01-28T16:51:12", tp3g.to_iso_string() );
-    //BOOST_CHECK_EQUAL( "20720128T165112", tp3g.to_non_delimited_iso_string() );
```
Comment (oxarbitrage) [inline tests/time_test.cpp:42]: please remove this 3 commented lines. thanks.

Comment (jmjatlanta) [reply tests/time_test.cpp:42]: Already done. See raw file.

**Review by jmjatlanta — APPROVED** [review]

File: tests/time_test.cpp (hunk 2 above)
Comment (oxarbitrage) [reply tests/time_test.cpp:42]: ops, seems you are right, sorry.

**Review by oxarbitrage — APPROVED** [review]
looks good to me too.

File: tests/time_test.cpp (hunk 2 above)
Comment (cogutvalera) [reply tests/time_test.cpp:42]: there aren't comments, removed already, look last commit please

**Comment by cogutvalera** [comment]
Thanks guys !

**Pull request merged**